src/status_im/chat/handlers/input.cljs

```diff
@@ -243,7 +246,7 @@
                      metadata
                      args]
               :as   content} :content
-             :keys     [chat-id jail-id data-type after]}]]
+             :keys     [chat-id jail-id data-type event-after]}]]
```

Comment:
It's a bit not clear that `event-after` is rather builder of event vector then event itself. It could be names `build-event-after`, `event-after-builder` or maybe someone have better ideas. But anyway, IMO better to emphasize somehow that this is not an event.